### PR TITLE
Integrating Foreman with Event-Driven Ansible

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: _posts/2023-03-21-integrating-foreman-with-event-driven-ansible.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+[user@eda ~]$ export JAVA_HOME=/usr/lib/jvm/jre-17-openjdk
+[user@eda ~]$ python3.9 -m venv ./eda-venv
+[user@eda ~]$ . ./eda-venv/bin/activate
+(eda-venv) [user@eda ~]$ pip install ansible-rulebook ansible ansible-runner
+(eda-venv) [user@eda ~]$ ansible-galaxy collection install ansible.eda
+(eda-venv) [user@eda ~]$ pip install -r /home/user/.ansible/collections/ansible_collections/ansible/eda/requirements.txt
+```
+The only difference to the official guides is that we explicitly used Python 3.9, as the default Python in CentOS Stream 8 is Python 3.6 and that is too old for `ansible-rulebook` and `ansible-runner`.
+
+When we now try to run the `ansible-rulebook` CLI, it will fail due to a [bug in the 0.11.0 release](https://github.com/ansible/ansible-rulebook/issues/455):
```
Comment (evgeni) [inline _posts/2023-03-21-integrating-foreman-with-event-driven-ansible.md]: I've been told there will be a release soon, so if we publish the post *after* the release, we can avoid the odd hacks in here.

File: _posts/2023-03-21-integrating-foreman-with-event-driven-ansible.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+- foreman
+- ansible
+---
+
+[Event-Driven Ansible](https://www.ansible.com/use-cases/event-driven-automation) is a new way of automation where your automation can dynamically react to events that happen in your environment.
+Today, we want to show you how you can integrate that with Foreman using the [Foreman Webhooks plugin](https://github.com/theforeman/foreman_webhooks).
+
+<!--more-->
+
+As workflows and events one might want to react on will vary across different setups, we'll show one example integration and let everyone adopt it to their needs.
```
Comment (ehelms) [inline _posts/2023-03-21-integrating-foreman-with-event-driven-ansible.md]:
```suggestion
As workflows and events one might want to react to will vary across different setups, we'll show one example integration and let everyone adopt it to their needs.
```

**Review by ehelms — APPROVED** [review]
Conditional approval -- I think waiting for the release and updating the blog is the right direction.

**Comment by evgeni** [comment]
The release is out, I've updated the post. Merging.

**Pull request merged**